.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [10.x, 12.x, 14.x]
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Setup Node.js ${{ matrix.node }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+
+    - name: Cache Node.js modules
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
```

Comment:
What's this line for?

Couldn't spot it among the examples in https://github.com/actions/cache/blob/9c77c9dbfc97cd17e1add865174d4622362960f2/examples.md#node---npm :)

Reply:
Hehe, pondered the same thing before, this is from their example at https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-nodejs#example-caching-dependencies

I think it's a fallback if someone would happen to cache something without `-node-`, but I must admit I never fully understood it and then forgot about it

Reply:
Perhaps we should just remove the line? It will likely confuse a third person later on too :D

Reply:
You're probably right about the fallback, but as long as it's documented I'm willing to 🙈

Reply:
🙈